src/passes/Print.cpp

```diff
@@ -175,7 +175,7 @@ static std::ostream& operator<<(std::ostream& o, const SExprType& sType) {
     o << '(';
     auto sep = "";
     for (const auto& t : type) {
-      o << sep << t;
+      o << sep << SExprType(t);
```

Comment:
what does this fix?

Reply:
Compound reference types in tuple locals were being fully printed rather than referring to a heap type by name. After this PR, inline heap types like that can no longer be parsed, so this change is necessary to use the type names instead.